<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>506</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Pi Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="baseSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,40,30">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="column1" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="top" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="timeText">
          <property name="font">
           <font>
            <pointsize>72</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white
}</string>
          </property>
          <property name="text">
           <string>11:28</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="secondsAndPeriod">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="secondsText">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white
}</string>
            </property>
            <property name="text">
             <string>56</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="periodText">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white
}</string>
            </property>
            <property name="text">
             <string>PM</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="dateText">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white
}</string>
        </property>
        <property name="text">
         <string>Monday, January 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="column1spacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="column2">
      <property name="maximumSize">
       <size>
        <width>257</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 128); /* Black with 50% opacity */
    border-radius: 10px; /* Rounded corners */
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="column3">
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 128); /* Black with 50% opacity */
    border-radius: 10px; /* Rounded corners */
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="currentTempText">
         <property name="font">
          <font>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
         </property>
         <property name="text">
          <string>13°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="feelsLikeText">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
         </property>
         <property name="text">
          <string>Feels like 9°</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="weatherText">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
         </property>
         <property name="text">
          <string>Overcast clouds</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="divider1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: white;
}</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="day1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="day1Text">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
           </property>
           <property name="text">
            <string>Today</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="day1prec">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>0%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="day1icon">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>Textlabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="day1hi">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>9°</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="day1lo">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>8°</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="divider2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: white;
}</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="day2">
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="day2Text">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
           </property>
           <property name="text">
            <string>Tue</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="day2prec">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>50%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="day2icon">
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="day2hi">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>12°</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="day2lo">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>11°</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="divider3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: white;
}</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="day3">
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="day3Text">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
           </property>
           <property name="text">
            <string>Wed</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="day3prec">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>80%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="day3icon">
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="day3hi">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>6°</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="day3lo">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background-color: rgba(0, 0, 0, 0);
    border-radius: 0px;
}</string>
             </property>
             <property name="text">
              <string>4°</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="divider4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: white;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
